src/llamacrawl/services/readers/firecrawl.py

```
@@ -634,11 +978,13 @@ async def cancel_crawl_job(self, job_id: str) -> bool:
 
         return success
 
-    async def get_crawl_status(self, job_id: str) -> dict[str, Any]:
-        """Get the status of a Firecrawl crawl job.
+    @async_retry_with_backoff()
+    async def get_crawl_status(self, job_id: str, auto_paginate: bool = True) -> dict[str, Any]:
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)